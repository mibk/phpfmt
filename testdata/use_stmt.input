<?php
namespace\Foo;
    use Key\Query;
use  ZZZ as A;
use   BB ;
use AAAA  ;
use League\Class;
use Meta\Http\Class;
use Meta;
use\DateTime;

class AA{}
