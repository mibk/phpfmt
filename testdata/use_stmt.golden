<?php

namespace \Foo;

use AAAA;
use BB;
use DateTime;
use Key\Query;
use League\Class;
use Meta;
use Meta\Http\Class;
use ZZZ as A;

class AA
{
}
